backends/tfhe-cuda-backend/src/cuda_bind.rs

```diff
@@ -129,6 +129,7 @@ extern "C" {
         stream: *mut c_void,
         gpu_index: u32,
         pbs_buffer: *mut *mut i8,
+        pbs_execution_mode: u32,
```

Comment:
There should be two scratches with pbs_execution mode no?

Reply:
Not anymore. I introduced and then removed that concept. That's the reason for the rebasing mess.

Reply:
Aaah ok